.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: go
 go:
-- 1.6.2
+- 1.7.1
```

Comment:
mackerel-agent is tested by 1.7,  https://github.com/mackerelio/mackerel-agent/blob/3a3d0b1a9c5d655fe21537c0115f9b4bfa092914/.travis.yml#L4
p-r: https://github.com/mackerelio/mackerel-agent/pull/266

I guess we should specify the same version, do you have a plan to change mackerel-agent line too?
Or I think it's ok to just specify 1.7.